components-registry-service-server/src/main/kotlin/org/octopusden/octopus/components/registry/server/service/impl/ComponentRegistryResolverImpl.kt

```diff
@@ -253,6 +258,23 @@ class ComponentRegistryResolverImpl(
         }
     }
 
+    private fun updateMetrics() {
+        LOG.info("Update metrics build system count")
```

Comment:
Let's created separate method for updating specific metric. In future new metrics will be added